src/scripts/scripts/zone/caverns_of_time/old_hillsbrad/boss_captain_skarloc.cpp

```diff
@@ -287,7 +287,7 @@ struct boss_captain_skarlocAI : public ScriptedAI
         else
             DevotionAura_Timer -= diff;
 
-        if (HeroicMode)
+        //Consecration (Heroic)
```

Comment:
idk if initial timer is 0 means he cast on pull? is -1 possible value or do we need heroic check?